src/main/java/training/subsystems/movement/Constants.java

```diff
@@ -0,0 +1,7 @@
+package training.subsystems.movement;
+
+public class Constants {
+    protected static final double MAX_ALLOWED_SPEED=0.9;
+    protected static final double RAD_FULL_CIRCLE_ANGLE=2*Math.PI;
+    protected static final double TALONFX_MOTOR_FULL_CIRCLE = 1;
```

Comment:
max allowed for what?
rad full circle is Rotation2d.fromRot(1)
motor rotation is 1 no need const